blaze-core/src/main/scala/org/http4s/blazecore/util/package.scala

```diff
@@ -1,12 +1,16 @@
 package org.http4s
 package blazecore
 
+import scala.concurrent.Future
 import fs2._
 
 package object util {
   /** Used as a terminator for streams built from repeatEval */
   private[http4s] val End = Right(None)
 
+  private[http4s] val FutureUnit =
+    Future.successful(())
```

Comment:
I'll bet there are a ton of these in blaze proper.  This is probably a constant that belongs down there, whence the futures come.

Reply:
There's `scala.concurrent.Future.unit`, but that might be a 2.12 thing.

Reply:
It is 2.12 only. :(